r-package/grf/R/local_linear_forest.R

```diff
@@ -222,11 +222,11 @@ predict.local_linear_forest <- function(object, newdata = NULL,
   if (!is.null(newdata) ) {
```

Comment:
I think it would be good to rename this to `ll_regression_forest` (and rename `tune_local_linear_forest` to `tune_ll_regression_forest`) to avoid confusion with LL causal forests.

Reply:
Okay that makes sense! Done :)

Reply:
Also, a longer follow-up: we'll do compute variance in a future PR! Tuning for ll.lambda will be my next PR and should be almost immediate, but I'd like to get a clean LL causal version pushed first since I need to write some tests for that.